react/week2/todo-list/src/Counter.js

```diff
@@ -0,0 +1,35 @@
+import React, {Component} from 'react';
+
+class Counter extends Component {
+  constructor(){
+    super()
+    this.state = {counter: 0}
+  };
```

Comment:
In the version of react we're using, you can omit the constructor and super, it's done behind the scenes, so you could re-write this like: 
```suggestion
    state = { counter: 0 };
```
You can read more about it in the [docs](https://reactjs.org/docs/react-component.html#constructor), however I will admit, they are a bit confusing to understand.

Reply:
it's week2 homework. 
implemented :)